package.json

```diff
@@ -1,16 +1,16 @@
 {
   "name": "ilib-resbundler",
-  "version": "1.0.2",
+  "version": "1.1.0",
   "description": "convert JSON type of resources to JS files",
   "main": "resbundler.js",
-  "bin":{
+  "bin": {
     "ilib-resbundler": "./resbundler.js"
   },
   "scripts": {
     "dist": "npm publish",
     "test-clean": "rm -rf test/result",
-    "test-assemble": "cd test;../resbundler.js -o result",
-    "test-dynamic": "cd test;../resbundler.js -a dynamic -o result -l 'ko-KR,es-ES,zh-Hans-CN'"
+    "test-assemble": "cd test;../resbundler.js --outDir=result",
+    "test-dynamic": "cd test;../resbundler.js --assembly=dynamic --outDir=result --locales=ko-KR,es-ES,zh-Hans-CN"
```

Comment:
Does it work with the equals sign? Or is it supposed to be like `--assembly dynamic` ?

Reply:
yes, it works. originally the reason why I want  to update code was because of following line.
 https://github.com/iLib-js/ilib-scanner/blame/master/README.md#L45 
I saw it says a `=` usage
 I tested both work.  --assembly=daynamic andn --assembley dynamic . 
 I think at least we have guide  same usage format ilib-sccanner and iilb-resbundler